membrane_rtc_engine/lib/membrane_rtc_engine/engine.ex

```diff
@@ -317,6 +317,14 @@ defmodule Membrane.RTC.Engine do
     Pipeline.call(rtc_engine, :get_endpoints)
   end
 
+  @doc """
+  Returns number of forwarded tracks in RTC Engine.
```

Comment:
Would be nice to add one sentence what we mean by forwarded tracks. E.g. it's not the number of published tracks but in fact, it's the number of active and pending subscriptions